### PR TITLE
[ci_gen_kustomize_values] Adding EDPM missing values and fixing SSH keys

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:

### CODE REVIEW

File: roles/ci_gen_kustomize_values/templates/edpm-values.j2 [hunk 1]
```diff
@@ -1,27 +1,28 @@
 data:
   ssh_keys:
-    authorized: |-
-{{ cifmw_ci_gen_kustomize_values_ssh_authorizedkeys | replace('\n', '\\n') | indent(6, first=True) }}
-    private: |-
-{{ cifmw_ci_gen_kustomize_values_ssh_private_key | replace('\n', '\\n') | indent(6, first=True) }}
-    public: |-
-{{ cifmw_ci_gen_kustomize_values_ssh_public_key | replace('\n', '\\n') | indent(6, first=True) }}
+    authorized: {{ cifmw_ci_gen_kustomize_values_ssh_authorizedkeys | b64encode }}
```
Comment (cjeanner) [inline roles/ci_gen_kustomize_values/templates/edpm-values.j2:3]: ah, we can use b64 here? that's even better!

Comment (pablintino) [reply roles/ci_gen_kustomize_values/templates/edpm-values.j2:3]: Yeah, they are: https://github.com/openstack-k8s-operators/dataplane-operator/blob/836a066e4b7c6ec1bc4f689f044484445a0f67a1/api/v1beta1/common.go#L81-L89

File: roles/ci_gen_kustomize_values/templates/edpm-values.j2 [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
-        - hostname: "{{ cifmw_ci_gen_kustomize_values_ntp_srv | default('pool.ntp.org') }}"
-      edpm_sshd_allowed_ranges:
-{% for range in cifmw_ci_gen_kustomize_values_sshd_ranges %}
-        - "{{ range }}"
-{% endfor %}
+      ansibleVars:
+        timesync_ntp_servers:
+          - hostname: "{{ cifmw_ci_gen_kustomize_values_ntp_srv | default('pool.ntp.org') }}"
+        edpm_network_config_os_net_config_mappings:
+          edpm-copmpute:
```
Comment (rebtoor) [inline roles/ci_gen_kustomize_values/templates/edpm-values.j2:18]: same old typo :D `s/copmpute/compute/g`

**Review by rebtoor — APPROVED** [review]

**Pull request merged**